app/src/main/java/com/github/palFinderTeam/palfinder/ui/login/LoginActivity.kt

```diff
@@ -66,7 +82,143 @@ class LoginActivity : AppCompatActivity() {
             startActivityForResult(signIntent, RC_GOOGLE_SIGN_IN)
         }
     }
```

Comment:
Not so important, but you may clean up unused code since if you want to revert back you can always use git.

Reply:
Yes, I will but some code here might be used if we implement the password-email login so I decided to leave if here for now, but it is definitly on my to do list